tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Settings.cs

```diff
@@ -10,11 +10,14 @@ public sealed class Settings
 
         public string OrganisationApiBaseUrl { get; }
 
+        public string BrokenOrganisationApiBaseUrl { get; }
+
         public Settings(IConfigurationRoot config)
```

Comment:
The type can be `IConfiguration` here (as we're not currently using either of the additional couple of things that `IConfigurationRoot` brings in.

Reply:
👍